t/0110-statfs-gotcha.t

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Source sharness environment scripts to pick up test environment
+# and UnifyFS runtime settings.
+#
+. $(dirname $0)/sharness.d/00-test-env.sh
+. $(dirname $0)/sharness.d/01-unifyfs-settings.sh
+
+# disable statfs from returning UnifyFS! super magic value, return tmpfs magic instead
+export UNIFYFS_CLIENT_SUPER_MAGIC=0
+
+$JOB_RUN_COMMAND $UNIFYFS_BUILD_DIR/t/sys/sysio-gotcha.t
```

Comment:
I think this is the only thing that needs to be fixed for the new `statfs` suite.
```suggestion
$JOB_RUN_COMMAND $UNIFYFS_BUILD_DIR/t/sys/statfs-gotcha.t
```

Reply:
Thanks @CamStan .  Nice catch!